allensdk/ipfx/data_set_features.py

```diff
@@ -171,24 +173,29 @@ def extract_cell_features(data_set,
         raise ft.FeatureError("Could not find hero sweep.")
 
     # short squares
+    logging.info("Analyzing Short Square")
     if len(short_square_sweep_numbers) == 0:
         raise ft.FeatureError("no short square sweep numbers provided")
 
     ssq_sweeps = data_set.sweep_set(short_square_sweep_numbers)
-    ssq_start, ssq_dur, _, _, _ = get_stim_characteristics(ssq_sweeps.sweeps[0].i, ssq_sweeps.sweeps[0].t)
+
+    ssq_start, ssq_dur, _, _, _ = st.get_stim_characteristics(ssq_sweeps.sweeps[0].i, ssq_sweeps.sweeps[0].t)
```

Comment:
I don't think you can assume that all short squares have the same start time anymore.  You need to align all of the timestamps so that they have the same estimation window.

Reply:
I would prefer to deal with alignment on a separate PR.